### DISCUSSION TITLE
Default wifi password not working

### DISCUSSION THREAD

**Discussion by capi92** [FYSETC/FYSETC-E4#9] [Q&A]
Hi,

Default wifi password is not working on my brand new e4 board.
Is there other default password than 12345678 ?
If not, is there a way to reflash default firmware to get it working or shall I send it back to the seller ?

Thanks for your help
Have a nice day
Pierre

**Comment by capi92** [comment]
Documentation : 
Turn on power to the FYSETC E4, and then connect your PC, smart phone, or tablet to the SSID "MARLIN_ESP" (default password is "12345678").

This is not working for me.

**Reply by xrenznaet** [reply]
try the following - using the command " **[ESP105] new_name_AP** " in the "Pronterface" or "Repeater Host", change the name of the access point and try to connect again with the default password. I managed.

**Comment by kaosbeat** [comment]
same here, not working.
What do you mean by the command [ESP105] new_name_AP? I got 
```
>>> [ESP105] new_name_AP
*** Unknown syntax: [ESP105] new_name_AP

```
back from pronterface. I'm connected fine to the serial interface
